### PR TITLE
Improve contributing guide, reference mentorship programs

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Subj, minor cleanup

### CODE REVIEW

File: content/community/contributing/index.md [hunk 1]
```diff
@@ -6,18 +6,33 @@ sidebar_multicard: false
...
 
+1. Join us on [Keptn Slack](/community/#slack).
+2. Join the `#help-contributing` channel.
+   It can be used for any questions related to Keptn contributions.
+   See the links to more specific channels, e.g. working groups, below.
+3. Explore the newcomer friendly issues, see the links below.
+
+## Keptn development
+
+If you are interested
```
Comment (StackScribe) [inline content/community/contributing/index.md:22]: Is something missing here?  Or perhaps this phrase is vestigal and should be deleted?

File: content/community/contributing/index.md [hunk 2]
```diff
@@ -6,18 +6,33 @@ sidebar_multicard: false
...
+1. Join us on [Keptn Slack](/community/#slack).
+2. Join the `#help-contributing` channel.
+   It can be used for any questions related to Keptn contributions.
+   See the links to more specific channels, e.g. working groups, below.
+3. Explore the newcomer friendly issues, see the links below.
+
+## Keptn development
+
+If you are interested 
+The most of Keptn codebase is implemented in Golang and JavaScript/Angular.
```
Comment (StackScribe) [inline content/community/contributing/index.md:23]:
```suggestion
Most of the Keptn codebase is implemented in Golang and JavaScript/Angular.
```

File: content/community/contributing/index.md [hunk 3]
```diff
@@ -6,18 +6,33 @@ sidebar_multicard: false
...
+We also heavily use Kubernetes, Helm, Docker and
+other container technologies like CloudEvents or OpenTelemetry.
```
Comment (StackScribe) [inline content/community/contributing/index.md:25]:
```suggestion
other container technologies like CloudEvents and OpenTelemetry.
```

File: content/community/contributing/index.md [hunk 4]
```diff
@@ -6,18 +6,33 @@ sidebar_multicard: false
...
+
+### Keptn core components
+
 **1️.** If you are new to Keptn and want to get started contributing, please take a look at our [**CONTRIBUTING.md**](https://github.com/keptn/keptn/blob/master/CONTRIBUTING.md) file that lays out the process on how to contribute to the project.
```
Comment (StackScribe) [inline content/community/contributing/index.md:29]:
```suggestion
**1️.** If you are new to Keptn and want to begin contributing, please take a look at our [**CONTRIBUTING.md**](https://github.com/keptn/keptn/blob/master/CONTRIBUTING.md) file that lays out the process on how to contribute to the project.
```
I wonder if "If you are new to Keptn" is necessary -- I could be an experienced Keptn user and still be new to contributing, right?  I think the sentence holds up fine without this.

Also, shouldn't this also refer to the. https://github.com/keptn/keptn.github.io/blob/master/CONTRIBUTING.md file for people who want to contribute to the documentation?

File: content/community/contributing/index.md [hunk 5]
```diff
@@ -6,18 +6,33 @@ sidebar_multicard: false
...
+We welcome new contributors to the Keptn project,
+regardless of your background and experience!
 
 ## Getting started
 
+1. Join us on [Keptn Slack](/community/#slack).
+2. Join the `#help-contributing` channel.
+   It can be used for any questions related to Keptn contributions.
+   See the links to more specific channels, e.g. working groups, below.
+3. Explore the newcomer friendly issues, see the links below.
```
Comment (StackScribe) [inline content/community/contributing/index.md:18]: Does "newcomer friendly issues" encompass anything more than the good-first-issue label?  If not, why introduce another name for it?

**Review by StackScribe — APPROVED** [review]
Very nice.  I made a couple suggestions for your consideration but all are non-blocking.

File: content/community/contributing/index.md (hunk 5 above)
Comment (oleg-nenashev) [reply content/community/contributing/index.md:18]: Agreed

```suggestion
3. Explore the good first issues, see the links below.
```

**Pull request merged**